source/about/accessibile_html_error_messages.md

```diff
@@ -1,58 +1,63 @@
 
-### What is Tex?
-TeX is a text formatting and markup language first released in late 1970s by [D. Knuth](https://en.wikipedia.org/wiki/TeX#Hyphenation_and_justification). It was created to properly configure and visualize mathematical formulae, tables and symbols in printed documents such as journals and books. 
+### What is LaTeXML?
 
-### What is LaTex?
-LaTeX is a TeX macro package developed in the early 1980s by [Leslie Lamport](https://en.wikipedia.org/wiki/LaTeX#cite_note-Lamport1986-4). It provides the ability to structure a document to meet different layout requirements such as section headers, footnotes, graphics and bibliographies.
+LaTeXML was developed at the [Digital Library of Mathematical Functions at NIST](https://math.nist.gov/) (National Institute of Standards and Technology) and is authored by Bruce Miller and Deyan Ginev. It's primary output is XML and a postprocessor can convert these XML documents into HTML with mathematical formulas as MathML. 
 
-### Why are TeX and LaTex  so popular? 
-They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty with. 
+### What is MathML?
+MathML is mathematical markup language and is an XML based application. MathML is used to describe and encode mathematical notation and scientific content for distribution on the web and other applications such as scientific software, print typesetting and voice synthesis. 
 
-TeX and LaTex are open source software programs and are supported, maintained and iterated upon by the communities that use them. As a result, there are many packages available offering a myriad of features to format research papers. 
+### Why are TeX and LaTeX so popular? 
+They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty rendering.
 
-TeX and LaTex packages are platform independent which allows for the ability to view documents regardless of the operating system or software set up making collaboration simple.
+TeX and LaTeX are open source software programs  that  are platform independent which allows for the ability to view documents regardless of the operating system or software set-up making collaboration simple. These programs are maintained and iterated upon by the communities that use them, and as a result, there are many packages available offering a myriad of features to format research papers. 
 
-### Why are TeX and LaTex so difficult to support?
-Due to the robust open source community supporting and developing TeX and LaTex packages, there are an enormous number of packages in use. We are continuously reviewing which features and packages are most popular in an effort to provide error free, accessible HTML papers.
+### Why are TeX and LaTeX so difficult to support?
```

Comment:
Why are we asking a question that suggests arXiv's whole strategy on this was a bad idea?

Reply:
At some point I recalled a discussion about why is it so challenging to support TeX and LaTeX. This was by no means meant to imply this strategy was a bad idea. I thought it more as managing expectations and indicating not all packages will be supported.